### PR TITLE
Added caption to multimedia & embedding images doc

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I added a caption to the Multimedia & Embedding Images doc. 
### Motivation

It will make the doc more accessible. 
### Related issues and pull requests

Relates to [#115](https://github.com/openwebdocs/project/issues/115)

### CODE REVIEW

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ slug: Learn/HTML/Multimedia_and_embedding/Images_in_HTML
 In the beginning, the Web was just text, and it was really quite boring. Fortunately, it wasn't too long before the ability to embed images (and other more interesting types of content) inside web pages was added. There are other types of multimedia to consider, but it is logical to start with the humble {{htmlelement("img")}} element, used to embed a simple image in a webpage. In this article we'll look at how to use it in depth, including the basics, annotating it with captions using {{htmlelement("figure")}}, and detailing how it relates to {{glossary("CSS")}} background images.
 
 <table>
+  <caption>Multimedia and Embedding Images</caption>
```
Comment (dawei-wang) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:11]:
```suggestion
 <caption>Multimedia and Embedding Images</caption> 
```

**Review by dawei-wang — COMMENTED** [review]
linting

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ slug: Learn/HTML/Multimedia_and_embedding/Images_in_HTML
 In the beginning, the Web was just text, and it was really quite boring. Fortunately, it wasn't too long before the ability to embed images (and other more interesting types of content) inside web pages was added. There are other types of multimedia to consider, but it is logical to start with the humble {{htmlelement("img")}} element, used to embed a simple image in a webpage. In this article we'll look at how to use it in depth, including the basics, annotating it with captions using {{htmlelement("figure")}}, and detailing how it relates to {{glossary("CSS")}} background images.
 
 <table>
+ <caption>Multimedia and Embedding Images</caption>
```
Comment (dawei-wang) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:11]:
```suggestion
<caption>Multimedia and Embedding Images</caption> 
```

**Comment by CBID2** [comment]
> linting

Still failed

**Comment by dawei-wang** [comment]
> > linting
> 
> Still failed

Try again with the new suggestion

**Comment by CBID2** [comment]
> > > linting
> > 
> > 
> > Still failed
> 
> Try again with the new suggestion

It failed again

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ slug: Learn/HTML/Multimedia_and_embedding/Images_in_HTML
 In the beginning, the Web was just text, and it was really quite boring. Fortunately, it wasn't too long before the ability to embed images (and other more interesting types of content) inside web pages was added. There are other types of multimedia to consider, but it is logical to start with the humble {{htmlelement("img")}} element, used to embed a simple image in a webpage. In this article we'll look at how to use it in depth, including the basics, annotating it with captions using {{htmlelement("figure")}}, and detailing how it relates to {{glossary("CSS")}} background images.
 
 <table>
+<caption>Multimedia and Embedding Images</caption>
```
Comment (dawei-wang) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:11]:
```suggestion
<caption>Multimedia and Embedding Images</caption>
```

Comment (dawei-wang) [reply files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:11]: there seemed to be a trailing at the end also

Comment (CBID2) [reply files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:11]: > there seemed to be a trailing at the end also

I see but I'm not sure where it's happening

Comment (CBID2) [reply files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md]: > 

It worked! :)

**Review by bsmth — APPROVED** [review]
Thanks @CBID2 👍🏻

**Pull request merged**